test/math/IntRandomGeneratorTest.ooc

```diff
@@ -51,6 +52,23 @@ IntRandomGeneratorTest: class extends Fixture {
 			expect(Int absolute(mean - expectedMean) < 2)
 			expect(Int absolute(deviation - expectedDeviation) < 2)
 		})
+		this add("uniform range", func {
+			uniformGenerator := IntUniformRandomGenerator new()
+			uniformGenerator setRange(-250_000, 250_000)
+			uniformLowest := 250_000
+			uniformHighest := -250_000
+			for (i in 0 .. 100_000) {
```

Comment:
I just learned about the `_` variable from @tomasmahlberg:
```ooc
for (_ in 0 .. 100_000) {
   // code that doesn't actually use `i` for anything
]
```
But I'm not going to ask you to change this now.